src/cli/argv_parser.js

```diff
@@ -111,6 +111,18 @@ export default class ArgvParser {
         ArgvParser.collect,
         []
       )
+      .option(
+        '--retry <NUMBER_OF_RETRIES>',
+        'specify the number of times to retry failing test cases (default: 0)',
+        parseInt,
+        0
+      )
+      .option(
+        '--retryWithTag <NUMBER_OF_RETRIES>',
```

Comment:
I think a better option would be `--retryTagFilter <TAG_EXPRESSION>`, specify what subset of scenarios can be retried.

Reply:
Makes sense. In that case the --retry option is still needed to specify the count of retrials, as I guess it can't or shouldn't be part of the tag expression?
If that's the case, what's the behaviour when --retryWithTag is specified but --retry is omitted?

Reply:
`--retry 4` -> Retries everything up to four times
`--retryTagFilter "@retry"` -> Does not retry anything
`--retryTagFilter "@retry" --retry 4` -> Retries @retry tests up to four times

Reply:
Yes, that's what I thought. However I would like the CLI to throw an error if user specifies --retryTagFilter without --retry itself. I don't like when CLI parameters are silently ignored.

Reply:
Sounds good on throwing an error if using retryTagFilter without retry

Reply:
Ok, I started to make those changes. Quite busy these days but I will try to do some work over the weekend. :)